change/change_test.go

```diff
@@ -14,3 +14,80 @@ func TestChange(t *testing.T) {
 	assert.Equal(t, "value", c.Value())
 	assert.Equal(t, uint64(1), c.Version())
 }
+
```

Comment:
Using some Benchmarks to justify the change from using `chan []*Change` to `chan []Change` semantics. Performance of value semantics are even better while preserving the inherent safety against nil pointer dereference